pkg/universe.dagger.io/python/image.cue

```diff
@@ -0,0 +1,13 @@
+package python
+
+import (
+	"universe.dagger.io/docker"
+)
+
+#Image: {
+	version: *"3.10-alpine" | string
```

Comment:
I liked what you did previously with the `alpine` field :)

Nit: You need to document public fields and definitions: https://docs.dagger.io/1226/coding-style#all-public-definitions-and-fields-must-be-documented

Reply:
I was considering something like:

```
#Image: {
	*{
		tag:  *"3.10-alpine" | string
		_tag: tag
	} | {
		version: *"3.10" | string
		alpine:  *true | false
		_tag: {
			if alpine {
				"\(version)-alpine"
			}
			if !alpine {
				"\(version)"
			}
		}
	}
}
```

That would allow the user to write `python.#Image & { version: "3.9", alpine: false }` or if you want something in particular: `python.#Image & { tag: "3.9.12-buster" }`.

Thoughts?

Reply:
addressed the nit in latest push. Happy to change to the implementation stubbed out above if you think it's better.

Reply:
I think it’s simpler to just use version & alpine. If you need more control over the tag you can replace `#Image` with your own `#Pull`.

Reply:
Updated to do that. Also added tests for this to test.cue which might be a little heavy.